comps/llms/vllm-openvino/README.md

```diff
@@ -0,0 +1,67 @@
+# Use vLLM with OpenVINO
+
+## Build Docker Image
+
+To build the docker image, run the command
+
+```bash
+bash ./build_vllm_openvino.sh
+```
+
+Once it successfully finishes you will have a `vllm:openvino` image. It can directly spawn a serving container with OpenAI API endpoint or you can work with it interactively via bash shell.
+
+## Use vLLM serving with OpenAI API
+
+### Start The Server:
+
+```bash
+bash launch_model_server.sh
+```
+
+For gated models such as `LLAMA-2`, you will have to pass -e HUGGING_FACE_HUB_TOKEN=\<token\> to the docker run command above with a valid Hugging Face Hub read token.
+
+Please follow this link [huggingface token](https://huggingface.co/docs/hub/security-tokens) to get the access token and export `HUGGINGFACEHUB_API_TOKEN` environment with the token.
```

Comment:
s/to get the access/to get an access

an because there are multiple tokens, not a single token, that will enable access. the is used for a singular entity.